merlin/systems/dag/ops/feast.py

```diff
@@ -19,8 +20,48 @@
 
 
 class QueryFeast(PipelineableInferenceOperator):
+    """
+    The QueryFeast operator is responsible for ensuring that your feast feature store [1]
```

Comment:
I think I'd initial cap "Feast" (Feast feature store) because it seems to be a project--we should treat it as a proper noun in that case.  When you mean "feature store", please type that out and there's no need to repeat Feast again because "feature store" is perfectly clear.  (This is like my "Triton Inference Server" the first time and "the server" afterward comment.)

"Triton Inference Server" in initial caps too, please.